lxd/storage/drivers/driver_zfs.go

```diff
@@ -264,12 +264,18 @@ func (d *zfs) Create() error {
 			}
 
 			d.config["source.wipe"] = ""
-		}
 
-		// Create the zpool.
-		_, err := shared.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], d.config["source"])
-		if err != nil {
-			return err
+			// Create the zpool.
+			_, err = shared.RunCommand("zpool", "create", "-f", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], d.config["source"])
```

Comment:
Just a nit, but to avoid duplication of the arguments (which can lead to bugs in the future if we change the invocation here as we need to remember to do it in two places now), I generally use something like this in these situations:

```go
args := []string{"create", "-m", "none", "-O", "compression=on"}

if shared.IsTrue(d.config["source.wipe"]) {
 args = append(args, "-f")
}

// Create the zpool.
args = append(args, d.config["zfs.pool_name"],d.config["source"])
_, err := shared.RunCommand("zpool", args...)
if err != nil {
    return err
}
```